src/app/compose/draftdesk.service.ts

```diff
@@ -309,6 +309,27 @@ export class DraftDeskService {
         this.draftModels.next(models);
     }
 
+    public async newTemplateDraft(
+        messageId: number,
+    ) {
+        forkJoin([
+            this.rmmapi.getMessageFields(messageId),
```

Comment:
You discovered a thing I didn't know existed..

I don't think we need getMessageFields here, as despite its name, getMessageContents actually stores the entire message object, see fetchMessageJSON in mailviewer/singlemailviewer.component.ts

Reply:
![2024-10-29-171952_611x250_scrot](https://github.com/user-attachments/assets/a2556adf-bbea-4860-b536-c0384229417e)

getMessageContents does not contain any fields. It is either the HTML or text of the message.

Reply:
Oo I see, the headers have that information. That is unusual to me.

Reply:
ah yeah the class doesnt contain alllll the fields (hysterical raisins no doubt)